Modules/Quotas/test/Quotas.Domain.Tests/QuotaIdTests.cs

```diff
@@ -0,0 +1,75 @@
+﻿using Backbone.Modules.Quotas.Domain.Aggregates.Identities;
+using FluentAssertions;
+using Xunit;
+
+namespace Backbone.Modules.Quotas.Domain.Tests;
+public class QuotaIdTests
+{
+    [Fact]
+    public void Can_create_quota_id_with_valid_value()
+    {
+        var validQuotaIdPrefix = "QUO";
+        var validIdLengthWithoutPrefix = 17;
+        var validIdValue = validQuotaIdPrefix + TestDataGenerator.GenerateString(validIdLengthWithoutPrefix);
+
+        var quotaId = QuotaId.Create(validIdValue);
+        var isQuotaIdValid = quotaId.IsSuccess;
+
+        isQuotaIdValid.Should().BeTrue();
+    }
+
+    [Fact]
+    public void Can_generate_quota_id()
+    {
+        var quotaId = QuotaId.Generate();
+        quotaId.Should().NotBeNull();
+    }
+
+    [Fact]
+    public void Cannot_create_quota_id_with_invalid_id_prefix()
+    {
+        var invalidQuotaIdPrefix = "QQQ";
+        var quotaId = QuotaId.Create(invalidQuotaIdPrefix + TestDataGenerator.GenerateString(17));
+
+        var isQuotaIdValid = quotaId.IsFailure;
```

Comment:
I once told you that I don't like that you create variables for everything that you assert on. This is an example of a problem that comes as a result. 😉 

```suggestion
        var isQuotaIdInvalid = quotaId.IsFailure;
```

Naming the variables wrong could cause confusion. Further you have to trace back where the variable is coming from when you want to debug a failing test. Whereas

```c#
quotaId.IsSuccess.Should().BeFalse();
```

is self explanatory.